bower.json

```diff
@@ -16,7 +16,7 @@
     "tests"
   ],
   "dependencies": {
-    "angular-material": "0.10.0",
+    "angular-material": "0.9.0",
```

Comment:
@tremainebuchanan This branch didn't have @matjames007 's fix for the white page issue when typing into the auto-complete text boxes. We had been using angular 9 on this one as a work-around.